iOS/PostEditor/RemoteChangePromptView.swift

```diff
@@ -9,6 +9,8 @@ struct RemoteChangePromptView: View {
     @Environment(\.horizontalSizeClass) var horizontalSizeClass
     @State private var promptText: String = "This is placeholder prompt text. Replace it?"
     @State private var promptIcon: Image = Image(systemName: "questionmark.square.dashed")
+    @State private var accessibilityLabel: String = "Replace"
+    @State private var accessibilityHint: String = "Replace this text with an accessibility hint"
```

Comment:
Does this text need to be updated to reflect the actual label and hint?

Reply:
It does! That happens in the `switch` case below (L35-36 and L40-41), which provides the logic for all of the prompt's content, including the icon, when the view is rendered.

Reply:
Got it! Thanks